source/grid/grid_out.cc

```diff
@@ -2461,13 +2464,68 @@ GridOut::write_svg(const Triangulation<2, 2> &tria, std::ostream &out) const
                                  (height -
                                   (height / 100.) * 2. * margin_in_percent))
                           << "\"/>" << '\n';
+
+
+                      if (svg_flags.label_boundary_id)
+                        {
+                          float distance_to_camera = std::sqrt(
+                            std::pow(point[0] - camera_position[0], 2.) +
+                            std::pow(point[1] - camera_position[1], 2.) +
+                            std::pow(point[2] - camera_position[2], 2.));
+                          float distance_factor =
+                            distance_to_camera /
+                            (2. * std::max(x_dimension, y_dimension));
+
+                          const auto font_size_this_edge =
+                            static_cast<unsigned int>(
```

Comment:
and you know the type for this variable: `unsigned int`

Reply:
Got it!